.github/workflows/posix.yml

```diff
@@ -62,8 +59,17 @@ jobs:
             pandas_testing_mode: "deprecate"
             test_args: "-W error"
             name: "Numpy Dev"
+          # Test lowest supported pyarrow
```

Comment:
Could you keep the above comment about not testing v2 or 3? It was a headache finding out that those version would stall the CI (The min version pandas actually supports its 1.0.1)